### PR TITLE
[analytics] Use window.ShopifyAnalytics to get service name

### PR DESCRIPTION
**Pull request by caropinzonsilva**
## ℹ️ What is the context for these changes?

...

## 🕹️ Demonstration

...

## 🎩 How can this be tophatted?

...

## ✅ Checklist

- [ ] Tested on mobile
- [ ] Tested on multiple browsers
- [ ] Tested for accessibility
- [ ] Includes unit tests
- [ ] Updated relevant documentation for the changes (if necessary)

### CODE REVIEW

File: packages/blockchain-components/src/utils/shopify/utils.test.ts [hunk 1]
```diff
@@ -1,84 +1,149 @@
...
 
-    describe('edge cases', () => {
-      it('returns OTHER for unknown services', () => {
-        expect(getShopifyService('/')).toBe(shopifyServices.OTHER.name);
-        expect(getShopifyService('/product-')).toBe(shopifyServices.OTHER.name);
-        expect(getShopifyService('/otherPathname')).toBe(
-          shopifyServices.OTHER.name,
-        );
+    describe('using pathname', () => {
+      describe("real shop's URLs", () => {
```
Comment (caropinzonsilva) [inline packages/blockchain-components/src/utils/shopify/utils.test.ts:63]: The diff is making it look like all the tests are new, but I just wrapped the previous tests in a `describe('using pathname')`.

File: packages/blockchain-components/src/utils/shopify/utils.test.ts [hunk 2]
```diff
@@ -1,84 +1,149 @@
...
+          expect(getShopifyServiceName(realShopUrls.commerceTown.page)).toBe(
+            shopifyServices.PAGE.name,
+          );
+        });
+      });
+
+      describe('edge cases', () => {
+        it('return Home for / pathname', () => {
+          expect(getShopifyServiceName('/')).toBe(shopifyServices.HOME.name);
+        });
```
Comment (caropinzonsilva) [inline packages/blockchain-components/src/utils/shopify/utils.test.ts:136]: This is also a new addition. If the pathname is `/`, we can assume that the service is the `HOME`. It might not work with some edge cases, but they will be easy to identify.

**Review by jamiely — APPROVED** [review]
Looks good, thanks!
